mods/player_api/init.lua

```diff
@@ -22,13 +20,5 @@ player_api.register_model("character.b3d", {
 
 -- Update appearance when the player joins
 minetest.register_on_joinplayer(function(player)
-	player_api.player_attached[player:get_player_name()] = false
 	player_api.set_model(player, "character.b3d")
-	player:set_local_animation(
-		{x = 0,   y = 79},
-		{x = 168, y = 187},
```

Comment:
was this moved? or removed?
why?

Reply:
Moved to `player_api.set_model`: With each model comes it's own set of animations.

Reply:
Ah, didn't see.